go/protocol/keybase1/extras.go

```diff
@@ -1619,25 +1619,20 @@ func (t TeamMembers) AllUIDs() []UID {
 	return all
 }
 
-func (t TeamMembers) AllUserVersions() []UserVersion {
-	m := make(map[UID]UserVersion)
+func (t TeamMembers) AllUserVersions() (res []UserVersion) {
 	for _, u := range t.Owners {
-		m[u.Uid] = u
+		res = append(res, u)
```

Comment:
res = append(res, t.Owners...)

(and for all the others below, instead of these for loops)